source/utilities.tex

```diff
@@ -17022,6 +17122,69 @@
 the same type, if any, as does \tcode{common_type_t<T2, T1>}.
 No diagnostic is required for a violation of this Note's rules.
 
+\pnum
+For the \tcode{common_reference} trait applied to a parameter pack \tcode{T} of
```

Comment:
We should add "Note C:" here and refer to it from the table, to be at least locally consistent. (Although, at a larger scale, we should move those "note-y" things out of the table and fully explain them outside of the table instead of having funny lettered notes which, adding insult to injury, are actually supposed to be normative.)